### DISCUSSION TITLE
' found conflicting values' when trying to train spaCy with custom tokenizer

### DISCUSSION THREAD

**Discussion by farrandi** [explosion/spaCy#8165] [Help: Coding & Implementations] [answered]
I am getting this error:
![image](https://user-images.githubusercontent.com/54917098/119059775-d4635280-b985-11eb-93c8-7290b2d1f3f5.png)

my config file has this at the end:

```
[initialize]
vectors = null
init_tok2vec = ${paths.init_tok2vec}
vocab_data = null
lookups = null
before_init = null
after_init = null

[initialize.components]

[initialize.tokenizer]

[initialize.before_init]
@callbacks = "customize_tokenizer"
```

and I called this in the command window:

    python3 -m spacy train config.cfg --code ./functions.py --output ./output[initialize]

Does anyone know why and how to fix this?

**Comment by polm** [comment, marked as answer]
I think the issue is that you gave two different values for `before_init`. If you remove the `before_init = null` line does it work?

**Reply by farrandi** [reply]
Oh yep that was it! Thank you!!
